src/core/or/circuitpadding.c

```diff
@@ -1160,7 +1160,7 @@ circpad_machine_schedule_padding,(circpad_machine_runtime_t *mi))
   tor_assert(mi);
 
   /* Don't schedule padding if we are currently in dormant mode. */
-  if (net_is_completely_disabled()) {
+  if (is_participating_on_network()) {
```

Comment:
This test is backwards.  We want to not schedule padding if we are _not_ participating on the network.